deployment/monocular/values.yaml

```diff
@@ -34,11 +34,16 @@ api:
       #- name: my-repo-name
       #  url: my-repository-url
       #  source: my-repository-source
+    cors:
+      allowed_origins:
+        - my-api-server
```

Comment:
I see this is a configurable parameter. Can you add some documentation? I'm not sure if the frontend will work with this default configuration.